### PR TITLE
Fix zip command in package-your-extension.md

### PR DESCRIPTION
**Pull request by mickro**
This MR fix given zip command to create extension package.

Fixes https://github.com/mozilla/extension-workshop/issues/624

### CODE REVIEW

File: documentation/publish/package-your-extension.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ contributors:
     david_ross,
     wbamberg,
     mrj,
+    mickro,
   ]
 last_updated_by: potterwrit
 date: 2019-07-15 1:14:39
```
Comment (championshuttler) [inline documentation/publish/package-your-extension.md:24]: Could you please update these lines as well? Thanks

Comment (mickro) [reply documentation/publish/package-your-extension.md:24]: which TZ are you using?

Comment (championshuttler) [reply documentation/publish/package-your-extension.md:24]: umm not sure , may be @caitmuenster can answer , but you can remove the time just date would be fine. Thanks

Comment (mickro) [reply documentation/publish/package-your-extension.md]: should be ok now

**Comment by willdurand** [comment]
Thanks!

File: documentation/publish/package-your-extension.md (hunk 1 above)
Comment (caitmuenster) [reply documentation/publish/package-your-extension.md]: That's good. Thanks @mickro! :)

**Comment by mickro** [comment]
Thank you @willdurand . See you next patch

**Pull request merged**